src/Core_Sideloader/UniversalAutoResolver/Core.UAR.Hooks.cs

```diff
@@ -257,7 +258,11 @@ internal static void ExtendedCoordinateLoad(ChaFileCoordinate file)
                     }
                 }
 
+#if AI || HS2                
+                IterateCoordinatePrefixes(ResolveStructure, file, extInfo, RetrieveSexOnClothes(file.clothes));
+#else
```

Comment:
Why not move the RetrieveSexOnClothes call inside IterateCoordinatePrefixes so all these #ifs are not needed?

Reply:
That can be done...